__tests__/author.test.js

```diff
@@ -0,0 +1,80 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const request = require('supertest');
+const app = require('../lib/app');
+
+describe('backend-express-template routes', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+  
+
+  it('#GET should return a list of authors', async () => {
+    const res = await request(app).get('/authors');
+    expect(res.status).toBe(200);
+    expect(res.body).toEqual([
```

Comment:
if you want to save yourself some space try doing this:  `expect(res.body).toEqual(expect.arrayContaining([{ insert an object}]);`